framework/basehandlers.py

```diff
@@ -331,6 +334,55 @@ def update_field_value(
     new_value = self.format_field_val(field, field_type, value)
     setattr(entity, field, new_value)
 
+  def update_stage(
```

Comment:
Why move this into base handlers?  It is called from more than one file?

Reply:
This is called from the OriginTrialsAPI class to update the OT stage, and also in the StagesAPI class to update general stages. I thought this might be appropriate to not duplicate the code here 🙂